### PR TITLE
Latest ember-qunit, @ember/test-helpers, ember-resolver

### PR DESCRIPTION
**Pull request by SergeAstapov**

### CODE REVIEW

File: test-app/config/ember-try.js [hunk 1]
```diff
@@ -11,6 +11,11 @@ module.exports = async function () {
         name: 'ember-lts-3.24',
         npm: {
           devDependencies: {
+            '@types/ember-qunit': '^5.0.0',
+            '@types/ember-resolver': '^5.0.11',
+            '@types/ember__test-helpers': '^2.6.1',
+            'ember-qunit': '^5.1.5',
+            'ember-resolver': '^8.0.3',
```
Comment (chriskrycho) [inline test-app/config/ember-try.js:18]: Note that these are *not* the latest versions—I'm good with the bump nonetheless, but we'll want to bump *further* as well.

**Review by chriskrycho — APPROVED** [review]

**Comment by chriskrycho** [comment]
1. CI failure is unrelated. Will put up a separate PR to fix it.
2. We should actually follow this up with *another* PR which bumps these to the *actual* latest and drops the `@types` packages. 🚀

File: test-app/config/ember-try.js (hunk 1 above)
Comment (chriskrycho) [reply test-app/config/ember-try.js:18]: Oh, I see – this is for the `ember-try` config for the old LTS. Reading fail!

**Comment by chriskrycho** [comment]
Correction on (2) above: we *do* have the latest; the actual change will be dropping 3.24 LTS from our support range (eventually).

**Comment by SergeAstapov** [comment]
@chriskrycho yeah, that was my assumption as well.

My idea with general purpose addons - if addon actually does work in older versions of Ember with no or minimum effort, there might be mo benefit dropping official support for older Ember versions. (this is highly opinionated of course)

**Pull request merged**